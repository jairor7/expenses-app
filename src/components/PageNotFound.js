import React, { Component } from "react";
import { Link } from "react-router-dom";

export default class PageNotFound extends Component {
  render() {
    return (
      <div>
        Page not found
        <Link to={"/"}>Go to home</Link>
      </div>
    );
  }
}
