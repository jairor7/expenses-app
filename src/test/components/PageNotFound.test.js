import React from "react";
import { shallow } from "enzyme";
import PageNotFound from "../../components/PageNotFound";
import toJSON from "enzyme-to-json";

test("should render PageJNotFound correctly", () => {
  const wrapper = shallow(<PageNotFound />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
