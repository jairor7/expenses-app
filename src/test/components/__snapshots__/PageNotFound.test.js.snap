// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render PageJNotFound correctly 1`] = `
<div>
  Page not found
  <Link
    to="/"
  >
    Go to home
  </Link>
</div>
`;
